modules/tinymce/src/themes/silver/main/ts/ui/dropdown/CommonDropdown.ts

```diff
@@ -162,6 +162,11 @@ const renderCommonDropdown = <T>(
           onControlAttached(spec, editorOffCell),
           onControlDetached(spec, editorOffCell)
         ]),
+        AddEventsBehaviour.config('common-button-display-events', [
```

Comment:
Should this be `menubutton-display-events` I think the name here is used for tooling.

Reply:
This also looks like a duplicate name with ToolbarButtons